word_way/scrapping/word.py

```diff
@@ -16,14 +16,28 @@
 from word_way.models import Pronunciation, Sentence, Word, WordSentenceAssoc
 from word_way.utils import convert_word_part
 
-__all__ = 'save_word', 'save_word_task',
+__all__ = 'save_word', 'save_word_task', 'save_words_task',
 
 
 @celery.task
 def save_word_task(target_word: str):
     save_word(target_word, session)
 
 
+@celery.task
+def save_words_task():
+    """단어가 없는 발음을 가져와서 단어를 저장하는 테스크"""
+    q = session.query(Word).filter(Word.pronunciation_id == Pronunciation.id)
```

Comment:
```suggestion
    subquery = session.query(Word).filter(
        Word.pronunciation_id == Pronunciation.id
    )
```
길어지더라도 의미를 알 수 있는 변수명으로 하는 것은 어떤가염?